hubble/hubble.go

```diff
@@ -670,6 +671,25 @@ func (k *K8sHubble) Enable(ctx context.Context) error {
 		if _, err := k.client.CreateService(ctx, k.params.Namespace, metricsSvc, metav1.CreateOptions{}); err != nil {
 			return err
 		}
+
+		// Add hubble-metrics port to Cilium agent daemonset
+		k.Log("🔥 Patching Agent DaemonSet...")
+		patch := []byte(fmt.Sprintf(`{
```

Comment:
Shouldn't we only do this if `hubble.metrics.enabled` is true?

Reply:
It's inside the `if metricsSvc := k.generateMetricsService(); metricsSvc != nil` block - so it will happen only if `hubble-metrics` service is created, so only if `hubble.metrics.enabled` is non-empty. I think tying this patch together with the service creation makes sense.